tests/ui/debuginfo/debuginfo-inline-callsite-location-macro-1.rs

```diff
@@ -0,0 +1,4124 @@
+//@ build-pass
+//@ compile-flags: -g -O
+//@ ignore-msvc
+// ignore-tidy-filelength
+
+#[derive(Default)]
+struct Something {}
+
+#[collapse_debuginfo(yes)]
+macro_rules! collapsed_macro {
+    ($code:expr) => {
+        // Important to use a function here, so it can be inlined
+        // at the MIR stage.
+        Something::default()
+    }
+}
+
+fn collapsed_debuginfo() {
+    // LLVM allows 12 bits for encoding discriminators, so 4096+1 calls to a
+    // MIR-inlined function will exhaust it. But because we're using
+    // collapse_debuginfo(yes) for this macro, we shouldn't need discriminators
+    // at all.
+    collapsed_macro!(1);
```

Comment:
Question: can these macro calls be generated by another macro?

Reply:
As far as I know not without a proc-macro (e.g. what the seq_macro crate does).